layer_visualization/vector_layer_visualization.py

```diff
@@ -0,0 +1,530 @@
+#Import necessary layers
+import os
+import json
+import cartosql
+import pandas as pd
+import mapclassify
+from colour import Color
+import json
+import time 
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stderr, level=logging.INFO)
+
+
+# username and api key of the carto account 
+CARTO_USER = os.getenv('CARTO_WRI_RW_USER')
+CARTO_KEY = os.getenv('CARTO_WRI_RW_KEY')
+
+'''
+Set up parameters for dataset
+'''
+# name of table on Carto which you want to make the visualization
+# this should be a table name that is currently in use
+table_name = 'soc_039_rw1_out_of_school_rate_edit'
+
+# if the dataset has a timeline, set timeline = True
+# otherwise, set timeline = False
+timeline = True
+# if timeline is true, set the year for making the visualization
+# if timeline is false, the year variable wont't be used, but don't comment it out
+year = 2018
+
+# geometry type
+geo_type = 'polygon' # 'polygon', 'line', or 'point'
+
+# set break type
+# basic - if use only one color
+# unique - if the colors are determined by unique values
+# choropleth - if use gradient colors determined by values
+break_type = 'choropleth' # 'basic', 'unique', or 'choropleth'
+
+# number of unique values/number of gradient breaks
+# if break_type is 'basic', the num_break variable wont't be used, but don't comment it out
+num_break = 5 
+
+# set colors
+# if break_type is 'basic', you have to manually set the color
+# colors = [''] # e.g. '#ffc0cb'
+# if break_type is 'unique' or 'choropleth', you can manually set the colors by the list, or use the function below to set the color ramp
+# colors = [''] # e.g. '#ffc0cb'
+colors = [color.hex_l for color in list(Color("white").range_to(Color("pink"), num_break))]
```

Comment:
Are there default palettes that we can pull from? Any resources you could link to figure out how to use them?